Kickstarter-iOS/Views/Controllers/InterstitialViewController.swift

```diff
@@ -0,0 +1,35 @@
+import Library
+import Prelude
+import Prelude_UIKit
+import UIKit
+
+internal final class InterstitialViewController: UIViewController{
+  
+  @IBOutlet fileprivate var mailIcon: UIImageView!
```

Comment:
Hey just a heads up that we moved away from interface builder, some more context here: https://github.com/kickstarter/rfcs/pull/54